content/gettingstarted/index.adoc

```diff
@@ -16,129 +16,132 @@ toc::[]
-image::os-console.png[OpenShift console]
-{nbsp} +
+Make sure your cluster environment has Istio installed. Because you are following these instructions to install the latest Kiali, you should not install the Kiali that comes with Istio (i.e. do not use the Helm setting `kiali.enabled=true` when installing Istio).
 
-In this case it is `http://kiali-istio-system.192.168.64.13.nip.io`. In your case this could be a different IP.
+For details on how to install Istio, see the link:https://istio.io/docs/setup/[Istio Setup docs].
 
-You can also use the `oc` command to determine the base-url:
+=== Kiali Operator
 
-[source,bash]
-----
-oc get route -n istio-system -l app=kiali
-----
+Now install the latest release of Kiali using the Kiali Operator.
 
-== Getting started on Kubernetes
+==== What is the Kiali Operator?
 
-To deploy Kiali to your Kubernetes cluster you can do the following.
+First, a quick introduction of what the Kiali Operator actually is.
 
-=== Preparation
+The Kiali Operator is an implementation of a link:https://coreos.com/operators/[Kubernetes Operator]. It provides a method of packaging, deploying, and managing Kiali.
 
-Preparation is only needed if you are installing in Minikube. When installing in Minikube, ensure that you enable the Ingress add-on by executing the following command:
+The Kiali Operator runs inside the `kiali-operator` namespace and listens for the creation of a Kiali CR (custom resource) object. When it sees one created, it will install Kiali using the settings it finds in the CR object. To see all the different settings you can configure in a Kiali CR, see link:https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[this fully documented example Kiali CR yaml file]. A minimal Kiali CR that falls back to all defaults looks like this:
 
-[source,bash]
+[source,yaml]
 ----
-minikube addons enable ingress
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
 ----
 
-You can now install Istio if needed. See the link:https://istio.io/docs/setup/[Istio Setup docs] for details.
+==== Installing the Kiali Operator
 
-=== Install Kiali
+In the instructions below, you will be shown how to install the Kiali Operator using the link:https://github.com/kiali/kiali/blob/master/operator/deploy/deploy-kiali-operator.sh[Kiali Operator install script]. That install script has the option of also immediately installing a simple Kiali CR for you so you do not have to, thus making it easy to get started with Kiali by running a single command (this is, in fact, the install script's default behavior). If, however, you wish to install Kiali specifically configured with custom settings of your choosing, you can tell the script to skip the installation of a Kiali CR and just simply install the Operator. In this case, you would be responsible for writing the Kiali CR yaml file and creating it in your cluster, thus allowing you to tell the Operator to install Kiali with the settings you want.
```

Comment:
>In the instructions below, you will be shown how to install the Kiali Operator using the 

In the instructions below, you will install the Kiali Operator using the ...

Reply:
done